### PR TITLE
Add Finnish localized strings for Unit menu

### PR DESCRIPTION
**Pull request by pascalpp**

### CODE REVIEW

**Comment by pascalpp** [comment]
@lanurmi Can you review these strings? I just used Google translate 😊

File: Free Ruler/fi.lproj/MainMenu.strings [hunk 1]
```diff
@@ -118,3 +118,18 @@
...
+"z2p-dA-zcS.title" = "Yksikkö";
+
+/* Class = "NSMenuItem"; title = "Unit"; ObjectID = "iDP-2z-irv"; */
+"iDP-2z-irv.title" = "Yksikkö";
+
+/* Class = "NSMenuItem"; title = "Pixels"; ObjectID = "pYR-Ba-kKi"; */
+"pYR-Ba-kKi.title" = "Pikselit";
+
+/* Class = "NSMenuItem"; title = "Millimeters"; ObjectID = "B6Y-Hi-AkN"; */
+"B6Y-Hi-AkN.title" = "Millimetrejä";
```
Comment (lanurmi) [inline Free Ruler/fi.lproj/MainMenu.strings:132]: Should be: "Millimetrit"

File: Free Ruler/fi.lproj/MainMenu.strings [hunk 2]
```diff
@@ -118,3 +118,18 @@
...
+
+/* Class = "NSMenuItem"; title = "Inches"; ObjectID = "lt1-Hj-2TR"; */
+"lt1-Hj-2TR.title" = "Tuumaa";
```
Comment (lanurmi) [inline Free Ruler/fi.lproj/MainMenu.strings:135]: Should be: "Tuumat"

**Review by lanurmi — CHANGES_REQUESTED** [review]
> @lanurmi Can you review these strings? I just used Google translate blush

Google translate is notoriously bad with Finnish, though this time it at least guessed the right words, while forms of some were a bit off.

**Pull request merged**